app/components/wallet/transactions/WalletTransactionsList.js

```diff
@@ -121,8 +121,8 @@ export default class WalletTransactionsList extends Component<Props> {
 
     return (
       <div className={styles.component}>
-        {transactionsGroups.map((group, groupIndex) => (
-          <div className={styles.group} key={walletId + '-' + groupIndex}>
+        {transactionsGroups.map(group => (
+          <div className={styles.group} key={walletId + '-' + group.transactions.map(tx => tx.id).join('-')}>
```

Comment:
this adds some performance hit. Can we just use the hash of the first tx of every group? they should be unique

Reply:
good idea!